tests/quantization/test_fp8.py

```diff
@@ -20,7 +21,12 @@
 @pytest.mark.skipif(not is_quant_method_supported("fp8"),
                     reason="FP8 is not supported on this GPU type.")
 @pytest.mark.parametrize("model_id", MODELS)
-def test_model_load_and_run(vllm_runner, model_id: str):
+@pytest.mark.parametrize("force_marlin", [False, True])
+def test_model_load_and_run(vllm_runner, model_id: str, force_marlin: bool,
+                            monkeypatch) -> None:
+    if force_marlin:
+        monkeypatch.setenv("VLLM_FORCE_FP8_MARLIN", "1")
```

Comment:
is it just used for testing? if yes, please prefix it with `VLLM_TEST`, so that users don't need to care about it.

Reply:
It could be useful for people who want to force fp8 marlin on Ada Lovelace, but I think it doesn't hurt to specify it for testing to declare intent